tests/test_parameter_mapping.py

```diff
@@ -138,6 +138,13 @@ def test_partial_override(condition_df_2_conditions,
             sbml_model=sbml_model
         )
 
+        # Comparison with NaN containing expected results fails after pickling!
```

Comment:
also add a test with more than one core?

Reply:
Can do, although it's a bit hard to check anything besides not-crashing. Therefore, I didn't see too much value. If sequential gets a separate implementation as suggested above, then it would make more sense. Although I'd rather avoid that.

Reply:
yes, I was just saying check that single-core and two-core gives the same result, nothing more (could e.g. be that pool has a special treatment for 1 process)

Reply:
Ah, right, that would make sense.